### PR TITLE
Updates for AuthZ v1 (GEA-13602)

### PR DESCRIPTION
**Pull request by kenany**
Renamed "namespace" to "type" and changed endpoint paths to `/v1/`.

### CODE REVIEW

File: pangea-sdk/v3/service/authz/api.go [hunk 1]
```diff
@@ -37,68 +37,62 @@ type TupleCreateResult struct {
...
 // @description Write tuples. The request will fail if tuples do not validate against the schema defined resource types.
 //
-// @operationId authz_post_v1beta_tuple_create
+// @operationId authz_post_v1_tuple_create
 //
 // @example
 //
 //	rCreate, err := cli.TupleCreate(ctx, &authz.TupleCreateRequest{
 //		Tuples: []authz.Tuple{
 //			authz.Tuple{
```
Comment (pangea-andrest) [inline pangea-sdk/v3/service/authz/api.go:46]: Should this line be? It was removed from the example in `examples` folder

File: pangea-sdk/v3/service/authz/api.go [hunk 2]
```diff
@@ -179,27 +173,27 @@ type TupleDeleteResult struct {
...
 // @description Delete tuples.
 //
-// @operationId authz_post_v1beta_tuple_delete
+// @operationId authz_post_v1_tuple_delete
 //
 // @example
 //
 //	rDelete, err := cli.TupleDelete(ctx, &authz.TupleDeleteRequest{
 //		Tuples: []authz.Tuple{
 //			authz.Tuple{
```
Comment (pangea-andrest) [inline pangea-sdk/v3/service/authz/api.go:182]: Same comment above

**Review by pangea-andrest — APPROVED** [review]
Approved with comments

File: pangea-sdk/v3/service/authz/api.go (hunk 1 above)
Comment (kenany) [reply pangea-sdk/v3/service/authz/api.go:46]: It's optional.

File: pangea-sdk/v3/service/authz/api.go (hunk 2 above)
Comment (kenany) [reply pangea-sdk/v3/service/authz/api.go:182]: Optional.

**Pull request merged**